source/leuven/wice_advanced_guide.rst

```diff
@@ -108,6 +108,26 @@ For getting a compact overview of the current state of the cluster, execute
 ``slurmtop`` on any KU Leuven Tier-2 node. Use ``slurmtop --help`` to get to
 know the functionality.
 
+.. _wice_environment_propagation:
+
+Environment propagation
+-----------------------
+
+Slurm jobs now start in a clean environment which corresponds to your login
```

Comment:
This is nitpicking, but a variable like `$VSC_ARCH_LOCAL` will not have the same value as in the login environment. I am not sure if adding a remark about this would clarify things or just make them more complicated, so you can decide if you want to add it.

Reply:
That's only if you define "login environment" as "environment you would get upon logging in to a login node" ;). Here I meant "login environment" in the context of a job, so as "login environment when logging in to an allocated compute node". For now I think this is an OK description that does not overcomplicate it.